tests/testthat/test-forecast.R

```diff
@@ -13,62 +11,58 @@ test_that("forecast_info() returns a list", {
   # test case 1/2: no data_version_date
   forecast_info_json <- jsonlite::read_json("data/forecast-71.json")
   m <- mock(forecast_info_json)
-  testthat::with_mock("zoltr::get_resource" = m, {
-    exp_forecast_info <- jsonlite::read_json("data/forecast-71.json")
-    exp_forecast_info$forecast_model_url <- exp_forecast_info$forecast_model
-    exp_forecast_info$forecast_model <- NULL
-    exp_forecast_info$created_at <- lubridate::parse_date_time(
-      "2020-03-05T15:47:47.369231-05:00", DATE_TIME_TZ_FORMAT, exact=TRUE)
-    exp_forecast_info$issued_at <- lubridate::parse_date_time(
-      "2021-05-10T08:38:41.296500-04:00", DATE_TIME_TZ_FORMAT, exact=TRUE)
-    exp_forecast_info$time_zero$timezero_date <- as.Date("2011-10-02")
-    exp_forecast_info$time_zero$data_version_date <- NA
-    act_forecast_info <- forecast_info(zoltar_connection, "http://example.com/api/forecast/1/")
-    expect_equal(length(mock_calls(m)), 1)
-    expect_equal(mock_args(m)[[1]][[2]], "http://example.com/api/forecast/1/")
-    expect_is(act_forecast_info, "list")
-    expect_equal(act_forecast_info, exp_forecast_info)
-  })
+  local_mocked_bindings(get_resource = m)
```

Comment:
same here. I don't really understand what `mock` is doing here. I think you can just do?:
```r
local_mocked_bindings(get_resource = forecast_info_json)
```